main/views.py

```diff
@@ -31,6 +33,12 @@ def index(request, **kwargs):  # pylint: disable=unused-argument  # noqa: ARG001
             and (is_admin_user(request) or is_learning_path_editor(request)),
             "is_article_editor": is_admin_user(request),
         },
+        "posthog": {
+            "api_key": settings.POSTHOG_PROJECT_API_KEY,
+            "enabled": settings.POSTHOG_ENABLED,
+            "timeout": settings.POSTHOG_TIMEOUT_MS,
+            "bootstrap_flags": all_flags,
+        },
```

Comment:
Recently @shanbady did some work to remove environment settings from the django view. Instead, any env variables that the frontend needs can be injected via the webpack build: https://github.com/mitodl/mit-open/blob/87fa1ad3e93905a391ad7cbb18c09fb1ed771904/frontends/mit-open/webpack.config.js#L87-L99

This is part of some work to make the HTML static so that it can be served by a CDN.
- https://github.com/mitodl/hq/discussions/3646

Reply:
(`user` is still injected via django, Jon is doing some work to stop that and get user info via an api)

Reply:
I changed this to not depend on Django injection at all - however this does change how the flag bootstrapping works; instead of being able to pull from the durable cache present there, it'll use the settings in the `.env` file instead. The default is for it to look at items that begin with `FEATURE_` and this is configurable by setting `MITOPEN_FEATURES_PREFIX` (which will also change it in the Django app). It doesn't do any processing of the feature flag name other than stripping the prefix so it's up to us to make sure these agree between the app and the PostHog settings. (The code does normalize a Python "True" into a JavaScript true - other values are left as-is.)

Reply:
Could we remove the `posthog` HTML injection all together now?